avaframe/ana1Tests/rotationTest.py

```diff
@@ -0,0 +1,240 @@
+        DFA simulation dataFrame
+    resTypeList : list
+        list of result types available
+    flagMass : boolean
+        should the aimec mass analysis be done
+    refSimRowHash : str
+        row index of the reference simulation in the simDF
+
+    Returns
+    -------
+    simDF : dataFrame
+        DFA simulation dataFrame updated with the energy line test results
+    flagMass : boolean
+        should the aimec mass analysis be done (switched to true if it is an entrainment simulation)
+    """
+    # read energy line config. ToDo should we only read the DFAPath config and remove the energy line one?
+    energyLineTestCfgFile = pathlib.Path('ana1Tests', 'energyLineTest_com1DFACfg.ini')
+    energyLineTestCfg, modInfo = cfgUtils.getModuleConfig(com1DFA, fileOverride=energyLineTestCfgFile, modInfo=True)
+    inDir = avalancheDir / 'Outputs' / 'com1DFA'
+    outDir = avalancheDir / 'Outputs' / 'com1DFARotated'
+    fU.makeADir(outDir)
+    # get reference angle
+    simName = simDF.loc[refSimRowHash, 'simName']
+    relName = (simName.split('_'))[0]
+    thetaRef = float(relName[3:])
+    for simHash in simDF.index:
+        # rotate results to be able to proceede to the aimec analysis
+        simName = simDF.loc[simHash, 'simName']
+        # activate mass analysis if it is a entrainment simulation
+        # (note this can be forced to be true with the flag at the top of the runScript file)
+        if 'ent' in simName:
+            flagMass = True
+        # copy mass file
+        massFile = list(inDir.glob('mass_*' + simName + '*.txt'))
+        shutil.copy(massFile[0], outDir)
+        # rotate simulation results (will be the input for the AIMEC analysis)
+        rotateDFAResults(avalancheDir, simDF, simHash, resTypeList, thetaRef)
+
+        # make energy line analysis
+        resultEnergyTest, savePath = energyLineTest.mainEnergyLineTest(avalancheDir, energyLineTestCfg, simName, dem)
+        simDF.loc[simHash, 'zEnd'] = resultEnergyTest['zEnd']
+        simDF.loc[simHash, 'sEnd'] = resultEnergyTest['sEnd']
+        simDF.loc[simHash, 'runoutAngle'] = resultEnergyTest['runoutAngle']
+        simDF.loc[simHash, 'energyLinePlotPath'] = savePath
+    # now compare energy line results to the reference
+    for simHash in simDF.index:
+        simDF.loc[simHash, 'zDiff'] = simDF.loc[simHash, 'zEnd'] - simDF.loc[refSimRowHash, 'zEnd']
+        simDF.loc[simHash, 'sDiff'] = simDF.loc[simHash, 'sEnd'] - simDF.loc[refSimRowHash, 'sEnd']
+        simDF.loc[simHash, 'runoutAngleDiff'] = simDF.loc[simHash, 'runoutAngle'] - simDF.loc[refSimRowHash, 'runoutAngle']
+
+    return simDF, flagMass
+
+
+def rotateDFAResults(avalancheDir, simDF, simHash, resTypeList, thetaRef):
+    """Rotate the DFA results
+
+    Parameters
+    ----------
+    avalancheDir : pathlib path
+        path to avalanche directory
+    simDF : dataFrame
+        DFA simulation dataFrame
+    simHash : str
+        row index of the simulation to analyze
```

Comment:
No it is the row hash, because fU.makeSimFromResDF now changes the index (it is so that aimec always gets a different row hash even for 2 identical simulations living in different places)